Tasks/AzureResourceManagerTemplateDeploymentV3/main.ts

```diff
@@ -0,0 +1,44 @@
+import tl = require("azure-pipelines-task-lib/task");
+import path = require("path");
+
+import deployAzureRG = require("./models/DeployAzureRG");
+import resourceGroup = require("./operations/ResourceGroup");
+import managementGroup = require("./operations/ManagementGroup");
+import subscription = require("./operations/Subscription");
+import armResource = require("azure-arm-rest-v2/azure-arm-resource");
+import armManagementGroup = require("azure-arm-rest-v2/azure-arm-management-group");
+import armSubscription = require("azure-arm-rest-v2/azure-arm-subscription");
+
+function run(): Promise<void> {
+    var azureRGTaskParameters = new deployAzureRG.AzureRGTaskParameters();
```

Comment:
rename this class to something like 'ARMDeployTaskParameters'

Reply:
and the local variable `azureRGTaskParameters` as well

Reply:
Done